src/App.css

```diff
@@ -0,0 +1,34 @@
+
+@import 'react-toastify/dist/ReactToastify.css';
+
+.login-bg{
+background-image: url(../src/assets/bg3.png);
+background-size: cover;
+/* background-repeat: no-repeat; */
+background-position: center;
+}
+
+.login-form{
+    background-color: rgba(49, 89, 81, 0.9);
```

Comment:
In this project, we will need to implement at the end light/dark mode. Having hardcoded colors will make it difficult to implement this feature. We need to store the colors as css variables and use this variables through our application.

lik this for example:
.light{
--bg-primary:#111;
--bg-secondary#222;
--text-primary:#333
...
}

and this class to a parent element to make this colors accessible in all of our application, and when we need to implement the dark mode, we will make another class with similar names but different values, so changing the class will change all values instantly.

You can watch a video explaining how to implement this feature.